### ISSUE TITLE
Loading bar for download?

### ISSUE THREAD

**Issue by rcpeene** [dandi/dandi-archive#1364]
I use the dandiapi to download very large files which take dozens of minutes to download. Occasionally, the download has halted or my kernel has frozen for a variety of reasons unbeknownst to me, but I'm stuck waiting anyway. It would be rather useful to me if asset downloads had an accompanying loading bar.

**Comment by AlmightyYakob** [comment]
How are you downloading assets? Through the browser, or through the [CLI](https://github.com/dandi/dandi-cli)?

**Comment by rcpeene** [comment]
Downloading via the Python API on a Jupyter notebook.

**Comment by AlmightyYakob** [comment]
Yeah that's the CLI, this is the repo for the API. I'll transfer this issue to that repo.
